crates/sui-storage/src/write_ahead_log.rs

```diff
@@ -66,7 +66,6 @@ pub trait WriteAheadLog<'a, C> {
     ///   proceeding.
     ///
     ///   Err(e) => An error occurred.
-    #[must_use]
```

Comment:
Why remove this? I can't think of a case where you'd call begin_tx and not hold the result.

Reply:
`Result` is already a `#[must_use]` type. This annotation is unnecessary.